src/universal/modules/spinner/components/Spinner/Spinner.js

```diff
@@ -0,0 +1,102 @@
+          <path className={css(styles.leaf, styles.leaf5)} d="M5.9102,7.1553 C6.0952,7.0883 6.2792,7.0213 6.4682,6.9573 C6.7542,6.8613 7.0492,6.7733 7.3452,6.6873 C7.5292,6.6343 7.7112,6.5783 7.8992,6.5273 C8.2342,6.4393 8.5782,6.3613 8.9242,6.2853 C9.0862,6.2493 9.2432,6.2093 9.4082,6.1763 C9.9242,6.0713 10.4512,5.9783 10.9912,5.9003 C11.3892,5.8433 11.7932,5.8063 12.1962,5.7653 C14.3632,5.5393 16.5762,5.5213 18.7442,5.7453 C18.7482,5.7463 18.7522,5.7463 18.7562,5.7463 C18.7082,5.6903 18.6582,5.6403 18.6102,5.5853 C18.5042,5.4643 18.3952,5.3483 18.2882,5.2303 C18.0642,4.9843 17.8402,4.7453 17.6112,4.5133 C17.4922,4.3923 17.3732,4.2733 17.2542,4.1573 C17.0212,3.9313 16.7872,3.7153 16.5532,3.5053 C16.4432,3.4083 16.3362,3.3073 16.2262,3.2153 C15.8862,2.9223 15.5442,2.6433 15.2022,2.3903 C13.2322,0.9273 11.3432,0.1553 9.7352,0.1553 C8.9952,0.1553 8.3272,0.3243 7.7502,0.6563 C6.0362,1.6453 5.2172,4.0523 5.4632,7.3143 C5.6072,7.2583 5.7632,7.2103 5.9102,7.1553"></path>
+          <path className={css(styles.leaf, styles.leaf6)} d="M17.1484,2.6924 C17.3774,2.8944 17.6044,3.1084 17.8304,3.3254 C17.9674,3.4574 18.1054,3.5854 18.2414,3.7214 C18.4834,3.9644 18.7204,4.2204 18.9584,4.4804 C19.0734,4.6064 19.1894,4.7254 19.3034,4.8554 C19.6514,5.2504 19.9944,5.6584 20.3314,6.0874 C20.5764,6.3984 20.8044,6.7344 21.0414,7.0594 C21.6834,7.9444 22.3004,8.8604 22.8604,9.8294 C23.4034,10.7694 23.8994,11.7384 24.3404,12.7214 C24.3404,12.7214 24.3434,12.7284 24.3454,12.7324 C24.3694,12.6644 24.3874,12.5964 24.4114,12.5294 C24.4634,12.3744 24.5104,12.2194 24.5604,12.0644 C24.6614,11.7494 24.7554,11.4334 24.8424,11.1214 C24.8864,10.9604 24.9294,10.7984 24.9704,10.6394 C25.0504,10.3224 25.1224,10.0074 25.1864,9.6974 C25.2154,9.5564 25.2484,9.4164 25.2744,9.2774 C25.3574,8.8364 25.4274,8.4004 25.4754,7.9774 C25.8894,4.3794 25.0794,1.7124 23.2504,0.6564 C21.5364,-0.3286 19.0144,0.1764 16.3394,2.0044 C16.4644,2.1034 16.5864,2.2154 16.7104,2.3194 C16.8564,2.4414 17.0024,2.5644 17.1484,2.6924"></path>
+        </svg>
+      </div>
+    </div>
+  );
+};
+
+Spinner.propTypes = {
+  fillColor: PropTypes.oneOf(ui.paletteOptions),
+  styles: PropTypes.object,
+  width: PropTypes.number
+};
+
+const keyframesRotate = {
+  '0%': {
+    transform: 'rotate(0deg)'
+  },
+  '100%': {
+    transform: 'rotate(360deg)'
+  }
+};
+
+const timing = 100 / 6;
+const keyframesOpacity = {
+  '0%': {
+    opacity: '.25'
+  },
+  [`${timing}%`]: {
+    opacity: '1'
+  },
+  [`${timing * 2}%`]: {
+    opacity: '1'
+  },
+  '100%': {
+    opacity: '.25'
+  }
+};
+
+const animationDuration = '3s';
```

Comment:
For code clarity. You could put the definition for `timing`, `keyframesOpacity`, and `animationDuration` on lines one after another, and capitalize them so it is clear they are constants. You could also add a comment that would inform the reader that you'd have to change all three of these constants by some multiple in order for the animation loop to render correctly. In the future (where everything is better), you could make this multiple a prop, so that the spinner timing could be changed dynamically.